tests/unittest/benchmark_runner/common/oc/test_oc.py

```diff
@@ -7,58 +7,66 @@
 from benchmark_runner.common.oc.oc_exceptions import YAMLNotExist, LoginFailed, NodeNotReady
 
 
-def test_create_async_pod_yaml_not_exist():
-    """
-    This method create pod from not exist yaml
-    :return:
-    """
-    oc = OC()
-    with pytest.raises(YAMLNotExist):
-        oc.create_async(yaml=f'stressng1.yaml')
+@patch('benchmark_runner.common.oc.oc.SingletonOCLogin')  # patching login
+def test_create_async_pod_yaml_not_exist(mock_login):
+    oc = OC(kubeadmin_password="dummy-password")
+    with patch("os.path.exists", return_value=False):
+        with pytest.raises(YAMLNotExist):
+            oc.create_async(yaml='stressng1.yaml')
 
 
-def dummy_long_uuid(cls, *args, **kwargs):
-    # Your custom testing override
-    return 'bb2be20e-e4e8-57ed-a7c0-efde559b1ce2'
+# Dummy UUID for testing
+def dummy_long_uuid(self, workload):
+    return 'bb2be20e-92d5-4f10-a6ae-94f71f479d12'
```

Comment:
Out of curiosity, why did you change this?

Reply:
Why would a unit test failure require you to change a GUID?

Reply:
It is just a mock for [get_long_uuid](https://github.com/redhat-performance/benchmark-runner/blob/129df6a59120ec5e4e32b33667def8445e97c585/tests/unittest/benchmark_runner/common/oc/test_oc.py#L28)

Reply:
I understand, I just wonder why you changed it here as opposed to using the previous one.

Reply:
I have mocked `dummy_long_uuid` because it used by `dummy_short_uuid`

Reply:
Revert it back to ` bb2be20e-e4e8-57ed-a7c0-efde559b1ce2`